indico/modules/legal/templates/agreement.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'layout/base.html' %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row_static, form_fieldset %}
+
+{% block page_class %}fixed-width-standalone-text-page{% endblock %}
+
+{% macro tos_field_reference(tos, tos_url) -%}
+    {%- if tos_url -%}
+        <a href="/tos" target="_blank">
+    {%- endif -%}
+    {%- trans %}terms of service{% endtrans -%}
+    {%- if tos_url -%}
+        </a>
+    {%- endif -%}
+{%- endmacro %}
+
+{% macro pp_field_reference(privacy_policy, privacy_policy_url) -%}
+    {% if privacy_policy_url %}
+        <a href="/privacy_policy" target="_blank">
+    {%- endif -%}
+    {%- trans %}privacy policy{% endtrans -%}
+    {%- if privacy_policy_url -%}
+        </a>
+    {%- endif -%}
+{%- endmacro %}
+
+{% block content %}
+{{ form_header(form) }}
+<style class="TODO">
+    /* Leaving this here for some advice during the review. I'd expect it would be easy to create a
+     * form where the labels are skipped but the required * remains, and the submit button is on the
+     * right side. It doesn't seem so. What am I missing? It doesn't seem like I should need to use
+     * these styles to make it look pretty. */
```

Comment:
Highlighting this comment for your reviewing pleasure :)

Reply:
I think if you want non-standard form formatting you're way better off doing it in react. Anyway, for a first functioning version (to test all the logic etc) I think we can keep the suboptimal styling

Reply:
Well, I would expect this is a somewhat standard case, but apparently it hasn't come up yet. So just leave out the styles now? Here is what it looks like without:

<img width="809" alt="image" src="https://github.com/indico/indico/assets/607198/fa24cf61-6647-4c06-b86d-a4a5ced4e8af">

and with:

<img width="889" alt="image" src="https://github.com/indico/indico/assets/607198/2b9df13c-70ab-4ef2-b21d-4ffef4b5b961">

Reply:
I would get rid of the `*` tbh - it's pretty much clear that the field is required, and it looks kind of out of place...

Styling wise I think it look much better if the checkbox is close to the left.

Reply:
Oh, have a look at `skip_label=true` in the `form_row()` macro call. I think that should disable the `*` but still keep the normal checkbox label since it's handled elsewhere

Reply:
Yeah I did see `skip_label`, but it removes the `*` as well. If you'd like it removed, that makes things easier. I've found an elegant way under those circumstances. Thanks!